ClosedXML/ClosedXML.csproj

```diff
@@ -2,6 +2,7 @@
 
   <PropertyGroup>
     <TargetFrameworks>netstandard2.0;netstandard2.1</TargetFrameworks>
+    <Nullable>enable</Nullable>
```

Comment:
Netstandard2.0 has to use annotations nullable context, because otherwise we will get false positive warnings due to lack of nullable attributes (e.g. `string.IsNullOrWhitespace` and other methods from BCL is not taken into account).